mflowgen/glb_top/custom-init/outputs/pin-assignments.tcl

```diff
@@ -35,6 +35,7 @@ proc port_compare {a b} {
 
 set array_width $env(array_width)
 set num_glb_tiles $env(num_glb_tiles)
+set prr_size 4
```

Comment:
What does this mean?

Reply:
Place and route region? And this affects how many flush outputs there are?

Reply:
yep. All columns in the same region will get the same flush signal. I will also add comments here.